templates/base.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE "html">

```

Comment:
The `DOCTYPE` declaration should be `<!DOCTYPE html>` instead of `<!DOCTYPE "html">`. The quotes around `html` are unnecessary and incorrect.